### PR TITLE
Feature/containerize front end

### LINKED ISSUE #27: Add Frontend Dockerfile
Create Dockerfile to containerize the react website application.

### PR DESCRIPTION
**Pull request by mputnik**
Closes #27.
containerized front-end application

### CODE REVIEW

File: react-website/hooks/post_checkout [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```
Comment (khalidkanaan) [inline react-website/hooks/post_checkout]: Delete this file and hooks directory, no longer needed

Comment (Justinzoorob) [reply react-website/hooks/post_checkout]: Yes, you are right habibi

File: react-website/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM node:16.17.1-buster-slim
+
+RUN mkdir -p /home/app
+
+WORKDIR /home/app
+
+COPY . /home/app
+
+RUN mkdir -p /home/app/themes-dist
```
Comment (khalidkanaan) [inline react-website/Dockerfile]: It is not necessary to create the directories here. The bind mount should be able to create these directories for us.

File: react-website/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+RUN mkdir -p /home/app/wet-boew-dist
```
Comment (Justinzoorob) [inline react-website/Dockerfile]: Remove this also, bind mount handles this.

**Review by jjsoong — APPROVED** [review]
Looks good

**Review by khalidkanaan — APPROVED** [review]

**Pull request merged**